client/src/core/client/stream/tabs/Notifications/NotificationContainer.tsx

```diff
@@ -35,6 +36,9 @@ const NotificationContainer: FunctionComponent<Props> = ({
     return createdAtDate.getTime() <= lastSeenDate.getTime();
   }, [createdAt, viewer]);
 
+  // eslint-disable-next-line no-console
+  console.log(body);
```

Comment:
Just noting here that this needs to be removed still.

Reply:
this has been removed! 👍